people/joboet.toml

```diff
@@ -0,0 +1,5 @@
+name = "joboet"
+github = "joboet"
+github-id = 25721079
+email = 'jonasboettiger@icloud.com'
```

Comment:
@joboet I took this email address from your most recent commit. Please confirm. :)

Reply:
Yup, that's the one!